### PR TITLE
Query and update

### PR DESCRIPTION
**Pull request by MAleemH**
**I have done the following:**

- [x] Add a column `species` of type `string`.
- [x] Inserted new data.
- [x] Inside a transaction, updated the species column and then rollback.
- [x] Inside a transaction, added species names.
- [x] Inside a transaction, deleted all records and then rollback.
- [x] Inside a transaction, updated some other data.
- [x] Added queries of aggregate.

**Screenshots are given below:**
Screenshot (62)
Screenshot (61)
Screenshot (60)
Screenshot (59)

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -8,3 +8,59 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
...
+ROLLBACK;
+
+SELECT * FROM animals;
+
+BEGIN;
+
+UPDATE animals SET species='digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species='pokemon' WHERE species IS NULL;
+
+COMMIT;
```
Comment (AnselemOdims) [inline queries.sql]: - [ ] Good job working on this query, but please kindly ensure you verify that the change was made before committing.

File: queries.sql [hunk 2]
```diff
@@ -8,3 +8,59 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
...
+
+/*Delete record*/
+
+BEGIN;
+
+DELETE FROM animals;
+
+ROLLBACK;
+
+SELECT * FROM animals;
```
Comment (AnselemOdims) [inline queries.sql]: - [ ] Good job working on this query, but please kindly ensure you verify that the change was made before rolling back.

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @MAleemH , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by Ghiftee — APPROVED** [review]
## STATUS: APPROVED ♻️♻️
Hi @MAleemH, 

To highlight 👍🏽 
✔️ Required information is added
✔️ Screenshots are added
✔️ README is professional

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀  
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
- Your license is broken, I suggest that you fix it as it makes your documentation more professional.



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**